scripts/cm_test_all_sandia

```diff
@@ -666,28 +663,23 @@ elif [ "$MACHINE" = "vega90a_caraway" ]; then
   #   output description and success based only on build succes; build time output (no run-time)
 
   BASE_MODULE_LIST="cmake,<COMPILER_NAME>/<COMPILER_VERSION>"
-  ROCM520_MODULE_LIST="$BASE_MODULE_LIST,openblas/0.3.20/rocm/5.2.0"
+  ROCM520_MODULE_LIST="$BASE_MODULE_LIST,openblas/0.3.20"
```

Comment:
Is this still correct to call the variables `ROCM520_`?

Reply:
We can modify the name, the important thing is to keep a separate list for rocm/5.2.0 vs rocm/5.6.1+rocm/6.0.0, the tpl (i.e. openblas) supported with rocm/5.2.0 is different from the other rocm versions

Reply:
The name is fine, I was not sure if we were still having modules with rocm/5.2.0 but if we do then that's totally fine!